### DISCUSSION TITLE
Education email disabling :(

### DISCUSSION THREAD

**Discussion by ThartWay** [community/community#68503] [New to GitHub] [answered]
Hi! Could someone help me on how to transfer an education git hub repo to a personal?

https://github.com/ThartWayy/

**Comment by sreshtech** [comment, marked as answer]
Here are the steps to do it:

1. Log in to GitHub:
   Make sure you are logged in to your GitHub account where you have access to both the education repository and your personal account.

2. Access the Repository:
   Go to the education repository that you want to transfer to your personal account.

3. fork the Repository:
   Since you can't directly transfer a repository from an organization or education account to a personal account, you'll need to fork it to your personal account. To do this, click the "Fork" button at the top-right corner of the repository's page. This will create a copy of the repository in your personal account.

4. Clone the Repository (Optional):
   If you want to work on the repository locally, you can clone it to your computer using Git. Open your terminal or command prompt and run the following command, replacing `<your_username>` with your GitHub username:
   
   
   git clone https://github.com/<your_username>/<repository_name>.git
   

5. Change Remote URL (Optional):
   By default, the remote URL in the cloned repository will still point to the education organization. You can change it to your personal repository by running the following commands in the cloned repository's directory:

   bash
   git remote remove origin
   git remote add origin https://github.com/<your_username>/<repository_name>.git
   



   

Now, the education repository has been transferred to your personal account as a fork. You have full control over it, and you can make changes, manage collaborators, and set up repository settings as needed. Remember to update any links or references to the old education repository with the new one in your personal account.hope its helpful.

**Reply by ThartWay** [reply]
thank youu!
